### DISCUSSION TITLE
Moving To Ubuntu

### DISCUSSION THREAD

**Discussion by I-Sarma-I** [pocketbase/pocketbase#5391] [Q&A] [answered]
I did all my configuration on my local machine that is running windows and i wanted to deploy it to a server that is running Ubuntu can i just copy the "pb_data" directory over ?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, by default only the executable is platform specific.

If you are extending PocketBase with Go, to cross-compile for Linux you need to specify the `GOOS` and `GOARCH` env variables when building (_also make sure that `CGO_ENABLED` is disabled_), ex.:

```bash
GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build
```
